Mathlib/Analysis/NormedSpace/Star/GelfandDuality.lean

```diff
@@ -159,31 +159,33 @@ theorem gelfandTransform_isometry : Isometry (gelfandTransform ℂ A) := by
 /-- The Gelfand transform is bijective when the algebra is a C⋆-algebra over `ℂ`. -/
 theorem gelfandTransform_bijective : Function.Bijective (gelfandTransform ℂ A) := by
   refine' ⟨(gelfandTransform_isometry A).injective, _⟩
-  suffices (gelfandTransform ℂ A).range = ⊤ by
-    exact fun x => ((gelfandTransform ℂ A).mem_range).mp (this.symm ▸ Algebra.mem_top)
+  /- The range of `gelfandTransform ℂ A` is actually a `StarSubalgebra`. The key lemma below may be
+    hard to spot; it's `map_star` coming from `WeakDual.Complex.instStarHomClass`, which is a
+    nontrivial result. -/
+  letI rng : StarSubalgebra ℂ C(characterSpace ℂ A, ℂ) :=
```

Comment:
Why is this a `letI` as opposed to just a regular `let`?

Reply:
I guess no reason. Switched.